git_theta/git_utils.py

```diff
@@ -193,74 +195,66 @@ def remove_file(f, repo):
         repo.git.rm(f)
 
 
-def git_lfs_install():
-    """
-    Run the `git lfs install` command
+def get_file_version(repo, path, commit_hash):
+    try:
+        commit = repo.commit(commit_hash)
```

Comment:
I think you can do item access on the tree objects, like `repo.commit(commit_hash).tree[path]`